### PR TITLE
Sprint post

### PR DESCRIPTION
**Pull request by pfctdayelise**
I evidently screwed up my master branch a bit somehow. Tried a lot of git-pulling to get it to match upstream but without success. Nevertheless, the changes shown are those relating to the new blog post and nothing else, so I give up at this point.

### CODE REVIEW

File: content/posts/2016/pytest-development-sprint.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+* [incorrectly dropping brackets on display of assertions](https://github.com/pytest-dev/pytest/issues/925)
+* [rename getfuncargvalue to getfixturevalue](https://github.com/pytest-dev/pytest/issues/1625)
+* [warning if you use getfuncargvalue with parametrized fixtures](https://github.com/pytest-dev/pytest/issues/460)
+* [terminal newlines in failed test output](https://github.com/pytest-dev/pytest/issues/1553)
+* [escaping curly braces in a tox command doesn't work](https://bitbucket.org/hpk42/tox/issues/212)
+* [Tox shouldn't call pip directly to avoid shebang limitations](https://bitbucket.org/hpk42/tox/issues/66)
+* [missing evaluated value in report when asserting a boolean attribute](https://github.com/pytest-dev/pytest/issues/1503)
+* [improved determination of rootdir](https://github.com/pytest-dev/pytest/pull/1621)
+
+
```
Comment (nicoddemus) [inline content/posts/2016/pytest-development-sprint.md:73]: While you said that this is work started/finished on the sprint, perhaps it is worth to emphasize that this is not a complete list of features that will go into 3.0:

> It is worth noting that this is not an exhaustive list of all features that will go into pytest-3.0; a separate post after the release will be made with all major highlights.

What do you think?

**Comment by nicoddemus** [comment]
Nice @pfctdayelise! :smile:

File: content/posts/2016/pytest-development-sprint.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+In total [35 pull requests were merged](https://github.com/pytest-dev/pytest/pulls?utf8=%E2%9C%93&q=is%3Apr%20is%3Amerged%20updated%3A2016-06-20..2016-06-27%20) to pytest, and [at least 12](https://bitbucket.org/hpk42/tox/pull-requests/?state=MERGED) to tox, not to mention many others to plugins.
+
+|   | |
+| ------------- | ------------- |
+| ![Learning outside]({attach}images/sprint_workingoutside.jpg){.img-rounded} | ![Stickers 2.0]({attach}images/sprint_stickers.jpg){.img-rounded}   |
+| ![Meeting a Black Forest local]({attach}images/sprint_horse.jpg){.img-rounded}  | ![Hiking in the sunny Black Forest]({attach}images/sprint_hiking.jpg){.img-rounded}    |
+| ![Sampling the local delicacies]({attach}images/sprint_cake.jpg){.img-rounded} | ![Enjoying the beergarden]({attach}images/sprint_beergarden.jpg){.img-rounded} |
+
+
+We also spent some time designing the [stickers](https://twitter.com/pytestdotorg/status/745528947736092672) and [t-shirts](https://github.com/kvas-it/pytest-design/blob/master/tshirt_example.png). Surveys will be going out to backers who chose these perks in the fundraising campaign very soon.
```
Comment (The-Compiler) [inline content/posts/2016/pytest-development-sprint.md]: They got out last week FWIW :wink:

File: content/posts/2016/pytest-development-sprint.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+Thank you so much to everyone who supported the sprint. We have greatly expanded knowledge of pytest internals, and we're looking forward to bringing you pytest 3.0 very soon (aiming for before [EuroPython](https://ep2016.europython.eu/conference/talks/pytest-30))!
```
Comment (The-Compiler) [inline content/posts/2016/pytest-development-sprint.md]: Maybe also mention that we onboarded some more contributors as well?

**Comment by pfctdayelise** [comment]
ping

**Comment by nicoddemus** [comment]
I already gave it a :+1:, so it looks good to be merged. :grin: 

Waiting for @pelme I guess.

**Comment by pfctdayelise** [comment]
I assigned it to him, I didn't realise other people would see it, so I don't think he will mind if you beat him to it :)

**Comment by nicoddemus** [comment]
Hopefully @pelme will forgive me then! :fearful:

**Pull request merged**